src/app/modules/group/components/group-manager-list/group-manager-list.component.html

```diff
@@ -5,15 +5,20 @@
 <ng-container *ngIf="state === 'ready'">
```

Comment:
Tested on [this group](https://dev.algorea.org/branch/feature/group-administration-page-allow-removal/en/#/groups/by-id/49268055312628123;path=/details/managers)

When I don't have the rights to remove managers, I can select managers but do nothing with the selection:
![image](https://user-images.githubusercontent.com/16257159/132707230-21ae6fe1-d53f-4fbc-89a9-67763a46c336.png)

Apparently it works exactly the on the composition tab, I doubt this is an expected behavior.
I think it would be better to show selection only when actions are available, for both managers and group composition

Footnote: Since the group composition is not the subject of this PR, it could be fixed in another one I think, I'll create the issue

Reply:
Fixed